### ISSUE TITLE
Node Pathway: Provide descriptive link text

### ISSUE THREAD

**Issue by MaoShizhong** [TheOdinProject/curriculum#27819]
Links in our lessons should be sufficiently descriptive to improve accessibility. As per the [layout style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#links), link text should both concisely describe its purpose (or where the link will redirect a user), and must not include the words `this` nor `here`.

## Guidance

- Comment on this issue to let us know you'd like to get a section assigned to you. You can also ask for a specific one.
- Please wait until you are assigned by a maintainer before starting to work on this issue to avoid multiple people working on the same lessons.
- When you have been assigned a section, please refer to the [suggested link text changes spreadsheet](https://docs.google.com/spreadsheets/d/1E1OJqUfBODvjdFMNgFMY_GvOLrOzV4ujL9o7Nxzg2zk/edit?usp=sharing) for instructions on what changes need to be made for your assigned lesson(s).
  - For most of these, you should just be able to copy and paste the suggested text to replace the original text shown in the spreadsheet (which may be more than just the link itself). Any additional steps/clarifications will be noted by the appropriate link(s).
  - You are welcome to use a different wording from a given suggestion if you find a better one. If you do, you must ensure the link text is consistent if there are multiple instances of the same href in the same lesson.
  - You are *not* required to fix any unrelated, preexisting linting errors such as list styles/indentation. Existing linting errors may cause certain tests to fail when you open a PR. You can ignore this and a maintainer will let you know if further action is required.
- Make sure to read the [general contributing guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md) as well as the [curriculum repo contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md) before starting your first contribution.
- Please add the following to your PR in the Issues section: `Related to #XXXXX` , since this issue is divided into sections, **individual PRs should NOT INCLUDE the `closes` keyword.**

### Intermediate HTML and CSS

- [x] @Crn0 [Emmet](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-emmet) && [SVG](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-svg)
- [x] @Gallardo166 [More Text Styles](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-more-text-styles) && [More CSS Properties](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-more-css-properties) && [Advanced Selectors](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-advanced-selectors)
- [x] @T0nci [CSS Functions](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-css-functions) && [Frameworks and Preprocessors](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-frameworks-and-preprocessors)
- [x] @nithindante [Form Basics](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-form-basics) && [Form Validation](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-form-validation) && [Project: Sign-up Form](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-sign-up-form) && [Introduction to Grid](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-introduction-to-grid)
- [x] @sasidharan19 [Creating a Grid](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-creating-a-grid) && [Advanced Grid Properties](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-advanced-grid-properties) && [Using Flexbox and Grid](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-using-flexbox-and-grid)

### JavaScript

- [x] @jooo-lee [A Quick Review](https://www.theodinproject.com/lessons/node-path-javascript-a-quick-review) && [Factory Functions and the Module Pattern](https://www.theodinproject.com/lessons/node-path-javascript-factory-functions-and-the-module-pattern)
- [x] @gingkapls [Classes](https://www.theodinproject.com/lessons/node-path-javascript-classes)
- [x] @RichardAnthonySanchez [ES6 Modules](https://www.theodinproject.com/lessons/node-path-javascript-es6-modules)
- [x] @yuliana-r [Webpack](https://www.theodinproject.com/lessons/node-path-javascript-webpack) && [Linting](https://www.theodinproject.com/lessons/node-path-javascript-linting)
- [x] @EdMagal [What is ES6?](https://www.theodinproject.com/lessons/node-path-javascript-what-is-es6) && [Asynchronous Code](https://www.theodinproject.com/lessons/node-path-javascript-asynchronous-code) && [A Very Brief Intro to CS](https://www.theodinproject.com/lessons/javascript-a-very-brief-intro-to-cs)
- [x] @Ac394 [Recursive Methods](https://www.theodinproject.com/lessons/javascript-recursive-methods) && [Project: Recursion](https://www.theodinproject.com/lessons/javascript-recursion) && [Space Complexity](https://www.theodinproject.com/lessons/javascript-space-complexity)
- [x] @tyj916 [Common Data Structures and Algorithms](https://www.theodinproject.com/lessons/javascript-common-data-structures-and-algorithms) && [HashMap Data Structure](https://www.theodinproject.com/lessons/javascript-hashmap-data-structure) && [Project: Binary Search Trees](https://www.theodinproject.com/lessons/javascript-binary-search-trees)
- [x] @josephdanielm [Project: Knights Travails](https://www.theodinproject.com/lessons/javascript-knights-travails) && [Using Git in the Real World](https://www.theodinproject.com/lessons/javascript-using-git-in-the-real-world)
- [x] @SeanAverS [Testing Basics](https://www.theodinproject.com/lessons/node-path-javascript-testing-basics) && [More Testing](https://www.theodinproject.com/lessons/node-path-javascript-more-testing) && [Conclusion](https://www.theodinproject.com/lessons/node-path-javascript-conclusion)

### Advanced HTML and CSS

- [x] @mikeyCos [Transforms](https://www.theodinproject.com/lessons/node-path-advanced-html-and-css-transforms) && [Transitions](https://www.theodinproject.com/lessons/node-path-advanced-html-and-css-transitions)
- [x] @thatsthat [Keyframes](https://www.theodinproject.com/lessons/node-path-advanced-html-and-css-keyframes) && [Natural Responsiveness](https://www.theodinproject.com/lessons/node-path-advanced-html-and-css-natural-responsiveness) && [Project: Homepage](https://www.theodinproject.com/lessons/node-path-advanced-html-and-css-homepage)

### React

- [x] @mathdebate09 [Introduction to React](https://www.theodinproject.com/lessons/node-path-react-new-introduction-to-react) && [Setting Up A React Environment](https://www.theodinproject.com/lessons/node-path-react-new-setting-up-a-react-environment) && [Reducing State](https://www.theodinproject.com/lessons/node-path-react-new-reducing-state)
- [x] @strallia [React Components](https://www.theodinproject.com/lessons/node-path-react-new-react-components) && [What is JSX?](https://www.theodinproject.com/lessons/node-path-react-new-what-is-jsx) && [Keys in React](https://www.theodinproject.com/lessons/node-path-react-new-keys-in-react) && [More On State](https://www.theodinproject.com/lessons/node-path-react-new-more-on-state)
- [x] @Siriuszx [Project: CV Application](https://www.theodinproject.com/lessons/node-path-react-new-cv-application) && [Refs And Memoization](https://www.theodinproject.com/lessons/node-path-react-new-refs-and-memoization)
- [x] @jwaddell10 [How To Deal With Side Effects](https://www.theodinproject.com/lessons/node-path-react-new-how-to-deal-with-side-effects) && [Class Based Components](https://www.theodinproject.com/lessons/node-path-react-new-class-based-components) && [Component Lifecycle Methods](https://www.theodinproject.com/lessons/node-path-react-new-component-lifecycle-methods) && [Mocking Callbacks And Components](https://www.theodinproject.com/lessons/node-path-react-new-mocking-callbacks-and-components)

### NodeJS

- [x] @BuonicosII [Introduction to the Back End](https://www.theodinproject.com/lessons/nodejs-introduction-to-the-back-end) && [Introduction to Frameworks](https://www.theodinproject.com/lessons/nodejs-introduction-to-frameworks)
- [ ] @MarkoMTP [Introduction: What is NodeJS?](https://www.theodinproject.com/lessons/nodejs-introduction-what-is-nodejs) && [Debugging Node](https://www.theodinproject.com/lessons/nodejs-debugging-node)
- [x] @ca-de-so [Getting Started](https://www.theodinproject.com/lessons/nodejs-getting-started)
- [x] @roopxx [Introduction to Express](https://www.theodinproject.com/lessons/nodejs-introduction-to-express) && [Express 101](https://www.theodinproject.com/lessons/nodejs-express-101)
- [x] @guskirb [Express 102: CRUD and MVC](https://www.theodinproject.com/lessons/nodejs-express-102-crud-and-mvc) && [Express 103: Routes and Controllers](https://www.theodinproject.com/lessons/nodejs-express-103-routes-and-controllers) && [Express 104: View Templates](https://www.theodinproject.com/lessons/nodejs-express-104-view-templates) && [Express 105: Forms and Deployment](https://www.theodinproject.com/lessons/nodejs-express-105-forms-and-deployment)
- [x] @beMimg [Project: Mini Message Board](https://www.theodinproject.com/lessons/nodejs-mini-message-board) && [Deployment](https://www.theodinproject.com/lessons/nodejs-deployment)
- [x] @looonnng [Authentication Basics](https://www.theodinproject.com/lessons/nodejs-authentication-basics) && [Security Configuration](https://www.theodinproject.com/lessons/nodejs-security-configuration) && [Project: Members Only](https://www.theodinproject.com/lessons/nodejs-members-only)
- [x] @kamranzafar4343 [API Basics](https://www.theodinproject.com/lessons/nodejs-api-basics) && [Testing Database Operations](https://www.theodinproject.com/lessons/nodejs-testing-database-operations)
- [x] @ryanleslie1997 [API Security](https://www.theodinproject.com/lessons/nodejs-api-security) && [Conclusion](https://www.theodinproject.com/lessons/nodejs-conclusion)

### Getting Hired

- [x] @jbrace817 [Qualifying Leads](https://www.theodinproject.com/lessons/node-path-getting-hired-qualifying-job-leads) && [Project: Building Your Resume](https://www.theodinproject.com/lessons/node-path-getting-hired-building-your-resume) && [Preparing to Interview and Interviewing](https://www.theodinproject.com/lessons/node-path-getting-hired-preparing-to-interview-and-interviewing) && [Conclusion](https://www.theodinproject.com/lessons/node-path-getting-hired-conclusion)

**Comment by strallia** [comment]
I'd be glad to help out with this issue. Could I be assigned the React lessons "React Components && What is JSX? && Keys in React && More On State"?

**Comment by MaoShizhong** [comment]
Of course you can - assigned that lesson group to you, @strallia

**Comment by ted-miller92** [comment]
Hello, I would like to contribute to this issue as my first contribution. May I take "What is ES6? && Asynchronous Code && A Very Brief Intro to CS"?

**Comment by MaoShizhong** [comment]
@ted-miller92 Assigned you that section!

**Comment by Crn0** [comment]
Hello, Mao! I would like to be assigned to a section if possible.

**Comment by MaoShizhong** [comment]
@Crn0 assign you Emmet && SVG :rocket:

**Comment by mathdebate09** [comment]
Hey Mao, please assign me the section for "Introduction to React && Setting Up A React Environment && Reducing State"

**Comment by MaoShizhong** [comment]
@mathdebate09 assigned!

**Comment by gingkapls** [comment]
Hello Mao! I'd be glad to contribute to the lesson on [Classes](https://www.theodinproject.com/lessons/node-path-javascript-classes) under Javascript.

**Comment by MaoShizhong** [comment]
@gingkapls assigned!

**Comment by RichardAnthonySanchez** [comment]
Is the ES6 modules still available? If so, may I be assigned for that lesson? Thanks in advance!

**Comment by MaoShizhong** [comment]
@RichardAnthonySanchez All yours!

**Comment by roopxx** [comment]
Hey Mao, could you please assign me these lessons:
1. Introduction to Express && Express 101
2. Express 102: CRUD and MVC && Express 103: Routes and Controllers && Express 104: View Templates && Express 105: Forms and Deployment

**Comment by MaoShizhong** [comment]
@roopxx For now, we'll only be assigning one lesson group per person, so I'll assign you "Introduction to Express" and "Express 101" :rocket:

**Comment by yuliana-r** [comment]
@MaoShizhong happy to help with any lesson group!

**Comment by MaoShizhong** [comment]
@yuliana-r Since you're currently already assigned to #27781, I thought it probably makes sense to assigned you Webpack and Linting, since that'll allow you to kill two birds with one stone.
You can do everything in a single PR, and say it's `Related to #27810` *and* `Closes #27781` in the PR form.

**Comment by MaoShizhong** [comment]
Assigned @mikeyCos !

**Comment by MaoShizhong** [comment]
@mikeyCos That's my error - good catch. I've updated the spreadsheet so the URL is included in the replacement text.

**Comment by josephdanielm** [comment]
Hey @MaoShizhong, may I please be assigned "Project: Knights Travails && Using Git in the Real World" for my first contribution? How meta!

**Comment by MaoShizhong** [comment]
Go for it, @josephdanielm!

**Comment by T0nci** [comment]
I would like to complete any assignment!

**Comment by nithindante** [comment]
Hey, Mao! First time contributer here, I would like to be assigned to a section if possible.

**Comment by MaoShizhong** [comment]
Thanks, both!
Assigned @T0nci to [CSS Functions](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-css-functions) && [Frameworks and Preprocessors](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-frameworks-and-preprocessors)
Assigned @nithindante to [Form Basics](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-form-basics) && [Form Validation](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-form-validation) && [Project: Sign-up Form](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-sign-up-form) && [Introduction to Grid](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-introduction-to-grid)

**Comment by jooo-lee** [comment]
Hi Mao! Could I be assigned to the lessons "A Quick Review && Factory Functions and the Module Pattern" under the JavaScript section?

**Comment by MaoShizhong** [comment]
You certainly can be @jooo-lee!

**Comment by nithindante** [comment]
Hey, Mao! I just did my first contribution, I dont know whether its okay to get one, if it is I would like to be assigned to a section

**Comment by MaoShizhong** [comment]
Have at it, @tyj916!
@nithindante great enthusiasm! For now, since it's not an emergency issue, we'll keep it to a section per person.

**Comment by T0nci** [comment]
@MaoShizhong Should we completely remove the links where there are blank cells in "Replace with this text" in the spreadsheet?

**Comment by sasidharan19** [comment]
Hello Mao, want to make my first contribution. Can I take this section in Intermediate HTML and CSS "Creating a Grid && Advanced Grid Properties && Using Flexbox and Grid" ?

**Comment by MaoShizhong** [comment]
@sasidharan19 assigned you those three lessons!

**Comment by Ac394** [comment]
@MaoShizhong Hi Mao, feel free to assign me any of these

**Comment by mrzamin** [comment]
Hello @MaoShizhong I'd like to get assigned if possible.

**Comment by MaoShizhong** [comment]
@mrzamin [Testing Basics](https://www.theodinproject.com/lessons/node-path-javascript-testing-basics) && [Project: Testing Practice](https://www.theodinproject.com/lessons/node-path-javascript-testing-practice) && [More Testing](https://www.theodinproject.com/lessons/node-path-javascript-more-testing) && [Conclusion](https://www.theodinproject.com/lessons/node-path-javascript-conclusion) are all yours

**Comment by MaoShizhong** [comment]
@thatsthat I've assigned you to [Keyframes](https://www.theodinproject.com/lessons/node-path-advanced-html-and-css-keyframes) && [Natural Responsiveness](https://www.theodinproject.com/lessons/node-path-advanced-html-and-css-natural-responsiveness) && [Project: Homepage](https://www.theodinproject.com/lessons/node-path-advanced-html-and-css-homepage)

**Comment by Siriuszx** [comment]
I want to volunteer 🙂

**Comment by MaoShizhong** [comment]
@Siriuszx Assigned you to [Project: CV Application](https://www.theodinproject.com/lessons/node-path-react-new-cv-application) && [Refs And Memoization](https://www.theodinproject.com/lessons/node-path-react-new-refs-and-memoization)

**Comment by jwaddell10** [comment]
I'd like to assist as well

**Comment by MaoShizhong** [comment]
@jwaddell10 I've assigned you to [How To Deal With Side Effects](https://www.theodinproject.com/lessons/node-path-react-new-how-to-deal-with-side-effects) && [Class Based Components](https://www.theodinproject.com/lessons/node-path-react-new-class-based-components) && [Component Lifecycle Methods](https://www.theodinproject.com/lessons/node-path-react-new-component-lifecycle-methods) && [Mocking Callbacks And Components](https://www.theodinproject.com/lessons/node-path-react-new-mocking-callbacks-and-components)

**Comment by MaoShizhong** [comment]
@BuonicosII I've assigned you [Introduction to the Back End](https://www.theodinproject.com/lessons/nodejs-introduction-to-the-back-end) && [Introduction to Frameworks](https://www.theodinproject.com/lessons/nodejs-introduction-to-frameworks)

**Comment by MarkoMTP** [comment]
can I be assigned with introdicution to node.js

**Comment by MaoShizhong** [comment]
@MarkoMTP I've assigned you to [Introduction: What is NodeJS?](https://www.theodinproject.com/lessons/nodejs-introduction-what-is-nodejs) && [Debugging Node](https://www.theodinproject.com/lessons/nodejs-debugging-node)

@BuonicosII for now, we're still sticking with one lesson group per person, but do have a look at the open issues list for any that are looking for help and have not yet been assigned/claimed. We also have other repos that may have open issues looking for help that you may feel comfortable working on. You can comment on whatever you wish to be assigned if so.

**Comment by guskirb** [comment]
Hello! I'd like to be assigned to a task please.

**Comment by ca-de-so** [comment]
Hey, I'd like to volunteer as well, any section will do.

**Comment by MaoShizhong** [comment]
Thanks both for volunteering!

@ca-de-so, I've assigned you [Getting Started](https://www.theodinproject.com/lessons/nodejs-getting-started)

@guskirb, I've assigned you [Express 102: CRUD and MVC](https://www.theodinproject.com/lessons/nodejs-express-102-crud-and-mvc) && [Express 103: Routes and Controllers](https://www.theodinproject.com/lessons/nodejs-express-103-routes-and-controllers) && [Express 104: View Templates](https://www.theodinproject.com/lessons/nodejs-express-104-view-templates) && [Express 105: Forms and Deployment](https://www.theodinproject.com/lessons/nodejs-express-105-forms-and-deployment)

**Comment by Ac394** [comment]
> @Ac394 I've assigned you [Recursive Methods](https://www.theodinproject.com/lessons/javascript-recursive-methods) && [Project: Recursion](https://www.theodinproject.com/lessons/javascript-recursion) && [Space Complexity](https://www.theodinproject.com/lessons/javascript-space-complexity)

Hello Mao! I think that on the Javascript chapter "Project Recursion" (Excel Sheet "Javascript" -> cell 27) the full stop(.) is misplaced, shouldn't it be at the end of the sentence?

Moreover, the assignments on "Recursive methods" chapter are still using the old list system (1, 2, 3...) rather than the new one (1, 1, 1...). Do you want me to update this too?

**Comment by MaoShizhong** [comment]
@Ac394 Good spot on the full stop, my error. It's been placed at the end of the sentence now!
For `recursive_methods.md`, you can switch the new one `1. 1. 1.` 
In the PR, I'll assess any existing lint errors and see if they're best resolved in that PR or separately (if memory serves correctly, I think there might be a couple there that need a separate issue once we're done with it here).

**Comment by looonnng** [comment]
Hello Mao, I'd like to volunteer and help out, any assignment will be fine! :)

**Comment by MaoShizhong** [comment]
@looonnng I've assigned you to [Authentication Basics](https://www.theodinproject.com/lessons/nodejs-authentication-basics) && [Security Configuration](https://www.theodinproject.com/lessons/nodejs-security-configuration) && [Project: Members Only](https://www.theodinproject.com/lessons/nodejs-members-only)

**Comment by beMimg** [comment]
Hi, would like to try contributing for the first time. May I take the "Project: Mini Message Board && Deployment"?

**Comment by MaoShizhong** [comment]
@beMimg no problem, I've assigned you those two lessons.

**Comment by kamranzafar4343** [comment]
Hy, I would like to work on API Basics && Testing Database operation. Thank you

**Comment by MaoShizhong** [comment]
@kamranzafar4343 Assigned those lessons to you

**Comment by jbrace817** [comment]
Hi, I would like to contribute. I'll take the Getting Hired section.

**Comment by MaoShizhong** [comment]
@jbrace817 I've assigned that section to you

**Comment by ryanleslie1997** [comment]
Hi, I'm happy to help with the API Security and Conclusion under NodeJS.

**Comment by MaoShizhong** [comment]
Assigned you that lesson group @ryanleslie1997!

**Comment by EdMagal** [comment]
I would like to have a section assigned to me, if there are any available yet. Thanks a lot. :)

**Comment by MaoShizhong** [comment]
Sorry, @EdMagal. All lesson groups have already been assigned.

That being said, @ted-miller92 and @mrzamin just a courtesy check in to see if this is still something you wish to work on?

**Comment by ryanleslie1997** [comment]
Hi, line 35 for API security doesn't contain all of the text as shown (from the spreadsheet) on the website. Should I add in what's already there?

**Comment by ryanleslie1997** [comment]
I managed to figure it out.

**Comment by MaoShizhong** [comment]
@EdMagal would you be happy to take the following lessons?
[What is ES6?](https://www.theodinproject.com/lessons/node-path-javascript-what-is-es6) && [Asynchronous Code](https://www.theodinproject.com/lessons/node-path-javascript-asynchronous-code) && [A Very Brief Intro to CS](https://www.theodinproject.com/lessons/javascript-a-very-brief-intro-to-cs)

**Comment by EdMagal** [comment]
> @EdMagal would you be happy to take the following lessons? [What is ES6?](https://www.theodinproject.com/lessons/node-path-javascript-what-is-es6) && [Asynchronous Code](https://www.theodinproject.com/lessons/node-path-javascript-asynchronous-code) && [A Very Brief Intro to CS](https://www.theodinproject.com/lessons/javascript-a-very-brief-intro-to-cs)

@MaoShizhong surely. Thanks a lot.

**Comment by MaoShizhong** [comment]
In that case I'll swap the assignment for those lessons to you instead, due to no response from the original assignee.

**Comment by MaoShizhong** [comment]
@jwaddell10 Just a courtesy check to see if this is still something you wish to work on. Let me know if you have any issues!

**Comment by SeanAverS** [comment]
Hello. If there are any open sections still available I would like to take one. Thank you.

**Comment by jwaddell10** [comment]
Yes sorry! Had other priorities but should have time today to complete

**Comment by MaoShizhong** [comment]
@SeanAverS would you be able to take [Testing Basics](https://www.theodinproject.com/lessons/node-path-javascript-testing-basics) && [Project: Testing Practice](https://www.theodinproject.com/lessons/node-path-javascript-testing-practice) && [More Testing](https://www.theodinproject.com/lessons/node-path-javascript-more-testing) && [Conclusion](https://www.theodinproject.com/lessons/node-path-javascript-conclusion)?

**Comment by SeanAverS** [comment]
@MaoShizhong Yes that sounds good!

**Comment by MaoShizhong** [comment]
@SeanAverS reassigning those lessons to you due to no response from the original assignee
